### PR TITLE
Feature/uh 004 update clients

### PR DESCRIPTION
**Pull request by sebastian0327**
image
image
image

### CODE REVIEW

File: src/features/client/entities/client.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        .max(255, "City name is too long"),
+    country: z.string() 
+        .min(1, "Country name is required")
+        .max(255, "Country name is too long"),
+    phone: z.string()
+        .min(1, "Phone is required")
+        .max(255, "Phone is too long"),
+    corporateEmail: z.string()
+        .min(1, "Phone is required")
+        .max(255, "Phone is too long"),
```
Comment (dgop92) [inline src/features/client/entities/client.ts]: email does not have it's corresponding validations

File: src/features/client/pages/ClientUpdate/useUpadteClient.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  });
+
+  const queryClient = useQueryClient();
+  const mutation = useMutation({
+    mutationFn: (Data: ClientUpadteunityInput) => updateClient(id, Data),
+    onSuccess: () => {
+      queryClient.invalidateQueries({ queryKey: ["opportunities"] });
+
+      enqueueSnackbar(
+        "Client is updated successfully",
```
Comment (dgop92) [inline src/features/client/pages/ClientUpdate/useUpadteClient.ts]: i think it is "was updated"

**Review by dgop92 — CHANGES_REQUESTED** [review]
look at the comments

**Review by dgop92 — APPROVED** [review]
ok

**Review by ndelarans — APPROVED** [review]
Ok

**Review by AlejandroSalas1143 — APPROVED** [review]
OK

**Pull request merged**